LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright {yyyy} {name of copyright owner}
+   Copyright © 2019-2022 Dell Inc. or its subsidiaries. All Rights Reserved.
```

Comment:
For other repositories, We have added only the creation date. To make it uniform can we add only the creation date here ?